src/__tests__/pages/Selector/index.jsx

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import { shallow } from 'enzyme';
-import '../../../setupTests';
+import 'SetupTests';
 
-import Selector from '../../../pages/Selector';
+import Selector from 'Pages/Selector';
```

Comment:
You can create an index within pages to export the selector and then do something like:

import { Selector } from 'Pages';